vj4/handler/judge.py

```diff
@@ -44,6 +50,36 @@ class JudgeDataListView(base.Handler):
       datalist.append({'domain_id': did, 'pid': pid})
     self.json({'list': datalist, 'time': int(time.time())})
 
+@app.route('/judge/data/{rid}', 'data_detail')
+class JudgeDataDetailView(base.Handler):
+  # @base.require_priv(builtin.PRIV_READ_RECORD_CODE | builtin.PRIV_WRITE_RECORD)
+  @base.route_argument
+  @base.sanitize
+  async def get(self, *, rid: objectid.ObjectId):
```

Comment:
You shouldn't response with a constructed a vj1-compatible zip.

This format is only for backwards compatibility. You should use it only if the original data is in this format. If not, please design a new protocol.